packages/cli/interactive.ts

```diff
@@ -1,10 +1,14 @@
 import { ConfirmPrompt, SelectPrompt, TextPrompt } from "@clack/core";
 import ansiEscapes from "ansi-escapes";
-import logUpdate from "log-update";
+import { createLogUpdate } from "log-update";
 import { blue, bold, brandColor, dim, gray } from "./colors";
 import { cancel, newline, shapes, space, status } from "./index";
 import type { ChalkInstance } from "chalk";
 
+process.stdout.columns = 300;
```

Comment:
No, but it can be improved in a followup. We should find a way to get the actual column width, which we're not able to get right now because we're in a child process